### PR TITLE
[OSS] [gpt2-Family] FT button enablement

### PR DESCRIPTION
**Pull request by Anubha98**
Tested out gpt2 family including 4 models

  1. Run success
  2. Deployment success
  3. Able to infer
Note: all the defaults will be same as HF ones

Experiment: https://ml.azure.com/experiments/id/db8f256e-9fbc-4070-9a93-3f9131bc5a06?wsid=/subscriptions/ed2cab61-14cc-4fb3-ac23-d72609214cfd/resourceGroups/training_rg/providers/Microsoft.MachineLearningServices/workspaces/train-finetune-dev-workspace&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

### CODE REVIEW

File: assets/models/system/distilgpt2/spec.yaml [hunk 1]
```diff
@@ -114,4 +119,20 @@ tags:
...
+      Standard_NC24s_v3,
+      Standard_NC24rs_v3,
+      Standard_NC4as_T4_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC64as_T4_v3,
+      Standard_ND40rs_v2,
+      Standard_ND96asr_v4,
+    ]
+version: 13
```
Comment (BhavanaTumma) [inline assets/models/system/distilgpt2/spec.yaml:138]: Can you check the version on the repo and then increment accordingly please?

Comment (Anubha98) [reply assets/models/system/distilgpt2/spec.yaml]: Fixed

**Review by BhavanaTumma — APPROVED** [review]
Approved

File: assets/models/system/distilgpt2/spec.yaml [hunk 2]
```diff
@@ -114,4 +119,20 @@ tags:
...
+      Standard_NC6s_v3,
+      Standard_NC12s_v3,
+      Standard_NC24s_v3,
+      Standard_NC24rs_v3,
+      Standard_NC4as_T4_v3,
+      Standard_NC8as_T4_v3,
+      Standard_NC16as_T4_v3,
+      Standard_NC64as_T4_v3,
+      Standard_ND40rs_v2,
+      Standard_ND96asr_v4,
```
Comment (skanakamedal) [inline assets/models/system/distilgpt2/spec.yaml:136]: there will be no issues with this additional ',' at the end in yaml files right? Can once check by loading the yaml file? (This is there for all   finetune_compute_allow_list)

Comment (Anubha98) [reply assets/models/system/distilgpt2/spec.yaml:136]: there will be no issues, i have checked it.

**Review by skanakamedal — APPROVED** [review]

**Pull request merged**